packages/@aws-cdk/region-info/lib/region-info-token.ts

```diff
@@ -0,0 +1,30 @@
+import cdk = require('@aws-cdk/cdk');
+import { RegionInfo } from './region-info';
+
+/**
+ * A Token that rocudes a value from looking up a RegionInfo fact based on the resolution context's stack region.
```

Comment:
`rocudes`? is that a word?

Reply:
![](http://www.quickmeme.com/img/df/df2afe22c2c8b842bbb6c7edaad47669e954406adff1afb2760bd1f2bd5a47a7.jpg)